packages/arb-token-bridge-ui/src/util/AnalyticsUtils.ts

```diff
@@ -170,6 +170,7 @@ const fathomEventToEventId: { [key in FathomEventMap]: string } & {
   'Fast Bridge Click: Across': 'EZDV8TMY',
   'Fast Bridge Click: Synapse': 'SKUFXFQR',
   'Fast Bridge Click: Stargate': '6VZXVGEQ',
+  'Fast Bridge Click: Wormhole': '',
```

Comment:
need to create one on Fathom?

Reply:
Should we have a separate event for CCTP Bridges, since some may be both in the USDC-specific modal and the withdrawal modal?